src/style.css

```
@@ -0,0 +1,152 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card-apple--width {
+  width: 200px;
+}
+
+.card-apple {
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+}
+
+.card-apple__img {
+  display: block;
+  height: 134px;
+  padding: 0 3px;
+  margin-bottom: 40px;
+  width: 100%;
+}
+
+.card-apple__commodity {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card-apple__code {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+
+  color: #616070;
+
+}
+
+.card-apple__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+  background-position: center;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars__star--last-child {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:first-child,
+.stars--2 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n + 3),
+.stars--4 > .stars__star:nth-child(-n + 4),
+.stars--5 > .stars__star:nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card-apple__feedback {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+  margin-left: 17px;
+  align-items: center;
+
+}
+
+.card-apple__price {
+  font-family: inherit;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card-apple__world {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+
+}
+
+.card-apple__number {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+
+}
+
+.buttom {
+  display: flex;
+  margin: 0;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  justify-content: center;
+  text-transform: uppercase;
+  border: 1px solid #00acdc;
+
+  text-decoration: none;
+  color: white;
+
+  align-items: center;
+  font-size: 14px;
+  font-weight: 700;
+
+  line-height: 16px;
+  text-align: center;
+}
+
+.buttom--hover:hover {
```

Use only pseudo-class here
```suggestion
.buttom:hover {
```